src/patient-search/patient-search.component.tsx

```diff
@@ -1,10 +1,96 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { match } from "react-router";
+import { debounce } from "lodash";
+import styles from "./patient-search.component.css";
+import { performPatientSearch } from "./patient-search.resource";
+import PatientSearchResults from "./patient-search-result/patient-search-result.component";
 
 export default function PatientSearch(props: PatientSearchProps) {
-  return <section>The patient search!</section>;
+  const searchTimeout = 400;
+  const [searchResults, setSearchResults] = useState([]);
+  const [emptyResult, setEmptyResult] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    if (searchTerm) {
+      performPatientSearch(searchTerm, "full").then(response => {
```

Comment:
Effects should clean themselves up. In this case that would mean canceling the network request with an abort controller. This prevents memory leaks and also unnecessarily calling `setState` after a component is unmounted.